src/lib/hooks/useConnect/setupConnection.ts

```diff
@@ -86,17 +91,30 @@ export async function setUpConnection({
         logger?.info?.('SendbirdProvider | useConnect/setupConnection/configureSession', sessionHandler);
         newSdk.setSessionHandler(sessionHandler);
       }
-      // to check if code is released version from rollup and *not from storybook*
-      // see rollup config file
-      // @ts-ignore
-      if (IS_ROLLUP === IS_ROLLUP_REPLACE) {
```

Comment:
@AhyoungRyu, can you remove these two from rollup config?

Reply:
Yeah removed those unused variables too https://github.com/sendbird/sendbird-uikit-react/pull/551/commits/4568885ea6f59553358bafc46331432789494d93